pkg/daemon/ceph/osd/volume.go

```diff
@@ -127,27 +131,30 @@ func getLVPath(op string) string {
 }
 
 func updateLVMConfig(context *clusterd.Context) error {
-	testArgs := []string{"-oL", "sed", "-i", "-e", "s#udev_sync = 1#udev_sync = 0#", "/etc/lvm/lvm.conf"}
-	if err := context.Executor.ExecuteCommand(false, "", "stdbuf", testArgs...); err != nil {
-		return fmt.Errorf("failed ceph-volume. %+v", err) // fail return here as validation provided by ceph-volume
-	}
-	testArgs = []string{"-oL", "sed", "-i", "-e", "s#udev_rules = 1#udev_rules = 0#", "/etc/lvm/lvm.conf"}
-	if err := context.Executor.ExecuteCommand(false, "", "stdbuf", testArgs...); err != nil {
-		return fmt.Errorf("failed ceph-volume. %+v", err) // fail return here as validation provided by ceph-volume
-	}
-	testArgs = []string{"-oL", "sed", "-i", "-e", "s#use_lvmetad = 1#use_lvmetad = 0#", "/etc/lvm/lvm.conf"}
-	if err := context.Executor.ExecuteCommand(false, "", "stdbuf", testArgs...); err != nil {
-		return fmt.Errorf("failed ceph-volume. %+v", err) // fail return here as validation provided by ceph-volume
+
+	exists, err := util.FileExists(lvmConfPath)
```

Comment:
Do we really need to check if it exists? How about if we skip this check and just fail if `ioutil.ReadFile()` fails to open the file?

Reply:
Yes, that's right. ReadFile() should be able to check the error. Removing the `FileExists()` method. Thanks.